src/core/shape/attributes.js

```diff
@@ -106,55 +182,108 @@ p5.prototype.noSmooth = function() {
 };
 
 /**
- * Modifies the location from which rectangles and squares are drawn. By default,
- * the first two parameters are the x- and y-coordinates of the shape's upper-left
- * corner. The next parameters are its width and height. This is equivalent to
- * calling `rectMode(CORNER)`.
+ * Changes the way rectangles and squares are drawn.
```

Comment:
Similar comment here, How about 'Changes where rectangles and squares are drawn'? @nickmcintyre

Reply:
Updated!